client/src/app/components/stardust/Output.tsx

```diff
@@ -268,8 +268,14 @@ class Output extends Component<OutputProps, OutputState> {
          let addressType: number = 0;
 
          if (output.type === ALIAS_OUTPUT_TYPE) {
-             address = output.aliasId;
-             addressType = ALIAS_ADDRESS_TYPE;
+            const aliasId = !HexHelper.toBigInt256(output.aliasId).eq(bigInt.zero) ?
+            output.aliasId :
```

Comment:
Indentation looks wrong. But can we move this logic to a private function below, so we don't repeat the same code in two places ?

Reply:
Done.